docs/compilation.rst

```diff
@@ -40,8 +40,8 @@ set the following environment variables:
 * | CUDA_VERSION - CUDA toolkit version (11.8 and 12.4 are officially supported, 11.0,
     11.1, 11.2, 11.4, 11.5, 11.6, 11.7, 12.0, 12.1, 12.2, 12.3, 12.4, 12.5 and 12.6 are deprecated
     and may not work).
-  | The default is ``12.8``. Thanks to CUDA extended compatibility mode, CUDA 11.1, 11.2, 11.3, 11.4
-    11.5, 11.6, 11.7 and 11.8 wheels are named as CUDA 11.0 because it can work with the CUDA 11.0 R450.x driver
+  | The default is ``12.9``. Thanks to CUDA extended compatibility mode, CUDA 11.1, 11.2, 11.3, 11.4
+    11.5, 11.6, 11.7, 11.8 and 11.9 wheels are named as CUDA 11.0 because it can work with the CUDA 11.0 R450.x driver
```

Comment:
CUDA 11.9? I believe this whole sentence should be about 12.x, and not 11.x

Reply:
Mechanical change. Fixed.